aws-android-sdk-iot/src/main/java/com/amazonaws/services/iot/model/CreateScheduledAuditRequest.java

```diff
@@ -75,6 +75,13 @@
      */
```

Comment:
Just double check if its just moving the code to a different place in the same file.

Reply:
Looks like it was a code move.